<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogEquipos</class>
 <widget class="QDialog" name="DialogEquipos">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>786</width>
    <height>607</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Edición de equipo</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_17">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Nombre Largo</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="le_Nombre_Largo">
         <property name="sizePolicy">
          <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
           <horstretch>10</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Nombre Corto</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="le_Nombre_Corto"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_6">
       <item>
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>Nombre Mayúsculas</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="le_Nombre_Mayusculas"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0" colspan="2">
        <layout class="QVBoxLayout" name="verticalLayout_8">
         <item>
          <widget class="QLabel" name="label_6">
           <property name="text">
            <string>Año de fundación</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="le_anyo_fundacion"/>
         </item>
        </layout>
       </item>
       <item row="0" column="2">
        <layout class="QVBoxLayout" name="verticalLayout_7">
         <item>
          <widget class="QLabel" name="label_10">
           <property name="text">
            <string>Nº Socios</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="le_n_socios"/>
         </item>
        </layout>
       </item>
       <item row="1" column="0" colspan="2">
        <layout class="QVBoxLayout" name="verticalLayout_9">
         <item>
          <widget class="QLabel" name="label_7">
           <property name="text">
            <string>Estadio</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="le_Estadio"/>
         </item>
        </layout>
       </item>
       <item row="1" column="2">
        <layout class="QVBoxLayout" name="verticalLayout_10">
         <item>
          <widget class="QLabel" name="label_9">
           <property name="text">
            <string>Aforo</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="le_Aforo"/>
         </item>
        </layout>
       </item>
       <item row="2" column="0">
        <layout class="QVBoxLayout" name="verticalLayout_11">
         <item>
          <widget class="QLabel" name="label_11">
           <property name="text">
            <string>Presupuesto BBDD</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="le_presupuesto_bbdd"/>
         </item>
        </layout>
       </item>
       <item row="2" column="1" colspan="2">
        <layout class="QVBoxLayout" name="verticalLayout_12">
         <item>
          <widget class="QLabel" name="label_8">
           <property name="text">
            <string>Presupuesto en juego</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="le_presupuesto_juego"/>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_13">
       <item>
        <widget class="QLabel" name="label_12">
         <property name="text">
          <string>Presidente</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="le_presidente"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_14">
       <item>
        <widget class="QLabel" name="label_15">
         <property name="text">
          <string>Patrocinador</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="le_patrocinador"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_15">
       <item>
        <widget class="QLabel" name="label_13">
         <property name="text">
          <string>Proveedor</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="le_proveedor"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_16">
       <item>
        <widget class="QLabel" name="label_14">
         <property name="text">
          <string>Nombre corto en mayúsculas</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="le_nombre_corto_mayusculas"/>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_14">
     <item>
      <widget class="QTableView" name="tableView_jugadores">
       <property name="editTriggers">
        <set>QAbstractItemView::NoEditTriggers</set>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QTableView" name="tableView_entrenadores">
       <property name="editTriggers">
        <set>QAbstractItemView::NoEditTriggers</set>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QLabel" name="label_2">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: rgb(0, 0, 188);</string>
           </property>
           <property name="text">
            <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-weight:600; color:#ffffff;&quot;&gt;ESCUDO SIMULADOR&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="mybutton" name="btn_ESCUDO_SIM">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QLabel" name="label">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: rgb(0, 0, 188);
</string>
           </property>
           <property name="text">
            <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-weight:600; color:#ffffff;&quot;&gt;ESCUDO&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="mybutton" name="btn_ESCUDO">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btn_guardar">
           <property name="text">
            <string>Guardar</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btn_volver">
           <property name="text">
            <string>Volver</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>mybutton</class>
   <extends>QPushButton</extends>
   <header location="global">mybutton.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>le_Nombre_Largo</tabstop>
  <tabstop>le_Nombre_Corto</tabstop>
  <tabstop>le_Nombre_Mayusculas</tabstop>
  <tabstop>le_anyo_fundacion</tabstop>
  <tabstop>le_n_socios</tabstop>
  <tabstop>le_Estadio</tabstop>
  <tabstop>le_Aforo</tabstop>
  <tabstop>le_presupuesto_bbdd</tabstop>
  <tabstop>le_presupuesto_juego</tabstop>
  <tabstop>le_presidente</tabstop>
  <tabstop>le_patrocinador</tabstop>
  <tabstop>le_proveedor</tabstop>
  <tabstop>le_nombre_corto_mayusculas</tabstop>
  <tabstop>tableView_jugadores</tabstop>
  <tabstop>tableView_entrenadores</tabstop>
  <tabstop>btn_ESCUDO_SIM</tabstop>
  <tabstop>btn_ESCUDO</tabstop>
  <tabstop>btn_guardar</tabstop>
  <tabstop>btn_volver</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>btn_volver</sender>
   <signal>clicked()</signal>
   <receiver>DialogEquipos</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>668</x>
     <y>418</y>
    </hint>
    <hint type="destinationlabel">
     <x>370</x>
     <y>221</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btn_guardar</sender>
   <signal>clicked()</signal>
   <receiver>DialogEquipos</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>711</x>
     <y>479</y>
    </hint>
    <hint type="destinationlabel">
     <x>392</x>
     <y>303</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
